src/EditorFeatures/Core/GoToDefinition/AbstractGoToCommandHandler`2.cs

```diff
@@ -106,7 +105,7 @@ public bool ExecuteCommand(TCommandArgs args, CommandExecutionContext context)
         if (service == null)
             return false;
 
-        Contract.ThrowIfNull(document);
+        Roslyn.Utilities.Contract.ThrowIfNull(document);
```

Comment:
conflict between us and VSThreading with ConfigureAwait on Task.Yield.  so i removed our using, which meant having to qualify this.